specification/billing/resource-manager/Microsoft.Billing/stable/2020-05-01/billing.json

```diff
@@ -3460,6 +3460,150 @@
         }
       }
     },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/createBillingRoleAssignment": {
```

Comment:
>createBillingRoleAssignment [](start = 71, length = 27)

You are modifying an existing stable API version. As per [https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#versioning](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#versioning), this is not allowed and requires a version bump.

Reply:
We have added new apis to same version earlier in past 15 days: https://github.com/Azure/azure-rest-api-specs/pull/10585

Reply:
Yeah I think I will allow it given that we've created an expectation that this is ok.

---
In reply to: [499105356](https://github.com/Azure/azure-rest-api-specs/pull/11023#discussion_r499105356) [](ancestors = 499105356)

Reply:
Thank you! @majastrz